bin/dotd

```diff
@@ -68,13 +68,19 @@ def open_in_default_browser(url)
   end
 end
 
+def run_on(server_name, command)
+  # escape any double quotes in the command string
+  command = %Q["#{command}"]
+  system "ssh -t gateway.code.org ssh -t #{server_name} \"#{command}\""
```

Comment:
Hmm. I thought it did, because the script worked despite the content push step having nested quotes, but upon closer inspection it looks like what you have above is exactly what we'd get.

In other words, I'm not actually sure what's going on here. I'll investigate more.